modules/ROOT/pages/product-versioning.adoc

```diff
@@ -1,25 +1,34 @@
-= Bonita Versioning
+= Bonita Versions
 :description: Discover how Bonita manages its product and artifacts versions and how it can impact your projects
 
-Discover how Bonita manages its product and artifacts versions and how it can impact your projects
+{description}
 
-== Product Versions
+== Platform Versions
 
 After years of semantic versioning, Bonita now follows the calendar versioning: YYYY.#-mmdd, since version 2021.1:
 
 * YYYY is the year of the release
-* # is the release number. Two versions are released every year.
-* mmdd is the month and day of the release. A new maintenance version is made available every month. For example, the February version will be 2021.1-0205 as it will be released on the 5th. Keep in mind that all maintenance versions are cross-compatible (see the Technical Id chapter).
+* # is the release number of the "main" release
+* mmdd is the month and day of the "maintenance" release. 
+
+For example:
+
+* The last main version was 2021.1
+* The February maintenance version will be 2021.1-0205 as it will be released on the 5th. +
+
+Keep in mind that:
+
+* Two main versions are released every year
+* A new maintenance version is made available at least every month to our customers 
```

Comment:
This is not always true

Reply:
This is the commitment made to Charles. We are not saying the Maintenance versions will be issued every first Thursday anymore, but we must release new Maintenance versions at least every month.